CONTRIBUTING.md

```diff
@@ -1,27 +1,88 @@
 
-Welcome to the BIDS Starters Kit repository! We're excited you're here and want to contribute.  
+:tada::balloon::cake: **Welcome to the BIDS Starters Kit repository!** :cake::balloon::tada:
 
-These guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know by opening an [issue](https://github.com/INCF/bids-starter-kit/issues)!
+:dizzy::hatched_chick::sunny: *We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
+
+The point of this starter kit is to **welcome new users and contributors to the BIDS community**. We hope that these guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know through one of the many ways to [get in touch](#get-in-touch).
+
+## Table of contents
+
+* [Joining the BIDS community](#joining-the-community)
+* [Get in touch](#get-in-touch)
+* [Contributing through GitHub](#contributing-through-github)
+* [Where to start: wiki, code and templates](#where-to-start-wiki-code-and-templates)
+* [Where to start: labels](#where-to-start-labels)
+* [Make a change with a pull request](#making-a-change-with-a-pull-request)
+* [Recognizing contributions](#recognizing-contributions)
 
-Before you start you'll need to set up a free [GitHub](https://github.com/) account and sign in. Here are some [instructions](https://help.github.com/articles/signing-up-for-a-new-github-account/).
 
 ## Joining the community
 
-BIDS is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
-We therefore require that all contributions adhere to our [Code of Conduct](http://www.brainhack.org/code-of-conduct.html).
+BIDS - the [Brain Imaging Data Structure][bids] - is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
+
+We therefore require that all contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
+
+How do you know that you're a member of the BIDS community? You're here! You know that BIDS exists! You're officially a member of the community. It's THAT easy! Welcome! :joy::raised_hands:
+
+
+## Get in touch
+
+There are lots of ways to get in touch with the team maintaining the BIDS Starter Kit.
+
+* Our channel in the [BrainHack slack team][brainhack-slack-starterkit]
+  * Click [here][brainhack-slack-invite] for an invite to the slack workspace
+* Our [Gitter channel][gitter]
+* The [BIDS mailing list][bids-mailinglist]
+
+If you're here during summer 2018 :icecream::palm_tree:, you should reach out to our lovely [Google Summer of Code][gsoc] student [Patrick Park][patrick-github]. He'll be monitoring all the channels above and it would really help his project along if you said hello and passed along any feedback you have :purple_heart:.
```

Comment:
Just a personal note that I usually add when asking for feedback "Don't be shy, the newer you are the more valuable your feedback is". 
Of course experienced users' feedback is valuable too, but I think very new users sometimes feel they don't know enough to give feedback, but the opinions from their "infancy" are actually the most valuable.
Curious to hear your thoughts on this

Reply:
❤️ Love it. Please could you update this text with your version? I think it would be nice to have your voice in this section.